pkg/models/components.go

```diff
@@ -228,9 +230,9 @@ func GetComponents() (result []Components, err error) {
 
 	}
 
-	templates = []string{"kube-dns", "heapster", "monitoring-influxdb", "iam", "openpitrix"}
+	templates = []string{"kube-dns", "heapster", "monitoring-influxdb", "iam", "openpitrix","istio","kubesphere"}
 
-	namespaces := []string{KUBESYSTEM, OPENPITRIX}
+	namespaces := []string{KUBESYSTEM, OPENPITRIX,ISTIO,KUBESPHERE}
```

Comment:
define as a constant variable in constants.go would be better

Reply:
constant will be handle together,for now there are in self model